.github/renovate.json

```diff
@@ -1,19 +1,46 @@
 {
+  "extends": [
+    "config:base"
+  ],
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "enabledManagers": ["custom.regex"],
   "prConcurrentLimit": 50,
-  "prHourlyLimit": 20,
+  "prHourlyLimit": 50,
   "branchConcurrentLimit": 50,
+  "separateMinorPatch": true,
   "labels": ["renovate"],
+  "logLevelRemap": [
+    {
+      "matchMessage": "/^Custom manager fetcher/",
+      "newLogLevel": "info"
+    }
+  ],
   "packageRules": [
     {
+      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204"],
+      "matchUpdateTypes": [
+        "minor"
+      ],
+      "automerge": false,
+      "enabled": false
```

Comment:
why not enabled ?

Reply:
This is a bit tricky. If we enable it, using azure-cni as an example, we will see the following PRs.
1. containernetworking/azure-cni	minor	v1.5.32 -> v1.6.6
2. containernetworking/azure-cni	patch	v1.6.3 -> v1.6.6
3. containernetworking/azure-cni	patch	v1.5.32 -> v1.5.36

#2 and #3 are what we need because we want to auto-update the patch version.
By enabling the "minor", #1 will also be created. 
As we discussed last week, we don't want Renovate to update minor for us, otherwise the assignee needs to cancel this PR manually every time (It is every time because v1.5.32 will/should never be updated to v1.6.x). Instead we are relying on the component owner to update to a new minor version.

Reply:
As a result, there will be many noise PRs created. A comparison of without and with `minor` enabled.
![image](https://github.com/user-attachments/assets/59f7feb9-1a25-4b65-b417-e703f940a451)

Reply:
thanks for the explaination, make sure you add that to you TSG/ Training material ww

Reply:
ya. writing this section now.